### PR TITLE
Update testing for 'Assimp' package

### PR DESCRIPTION
**Pull request by rbsheth**
* I have checked that this pull request contains only
  `.travis.yml`/`appveyor.yml` changes or commits merged from the `pkg.template` branch. All other changes send
  to https://github.com/ruslo/hunter. **Yes**

* I have checked that no toolchains removed from CI configs, they are commented
  out instead so other developers can enable them back easily and to simplify
  merge conflict resolution. **Yes**

* I have checked that for every commented out toolchain there is a link to the
  broken CI build page or to the minimum compiler requirements documentation
  so other developers can figure out what was the problem exactly. **Yes**

* I have checked that for every enabled toolchain corresponding package passed
  all stages of update cycle: test/merge/upload/release. **Yes**

### CODE REVIEW

**Comment by rbsheth** [comment]
@ruslo This should change the Assimp toolchains that work/don't work on the newest version.

File: .travis.yml [hunk 1]
```diff
@@ -43,11 +43,10 @@ matrix:
...
         PROJECT_DIR=examples/Assimp
 
-    # FIXME: https://travis-ci.org/ingenue/hunter/jobs/313393099
-    # - os: linux
-    #   env: >
-    #     TOOLCHAIN=android-ndk-r17-api-24-arm64-v8a-clang-libcxx14
-    #     PROJECT_DIR=examples/Assimp
+    - os: linux
+      env: >
+        TOOLCHAIN=android-ndk-r17-api-24-arm64-v8a-clang-libcxx14
```
Comment (ruslo) [inline .travis.yml:48]: If you want to enable broken toolchain then you should submit new version first:
- https://docs.hunter.sh/en/latest/creating-new/update.html#add-toolchains

**Review by ruslo — CHANGES_REQUESTED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -86,11 +85,12 @@ matrix:
         TOOLCHAIN=osx-10-13-cxx14
         PROJECT_DIR=examples/Assimp
 
-    - os: osx
-      osx_image: xcode9.4
-      env: >
-        TOOLCHAIN=ios-nocodesign-11-4-dep-9-3
-        PROJECT_DIR=examples/Assimp
+    #FIXME: https://travis-ci.com/rbsheth/hunter/jobs/209825923
```
Comment (ruslo) [inline .travis.yml]: iOS is working fine, please revert:
- https://travis-ci.org/ingenue/hunter/builds/552984728

**Review by ruslo — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (rbsheth) [reply .travis.yml]: Fixed

**Review by ruslo — APPROVED** [review]

**Pull request merged**